### ISSUE TITLE
`git am` reported as REBASING

### ISSUE THREAD

**Issue by ilovezfs** [nojhan/liquidprompt#527]
Shell: bash
Operating system: macOS
Liquid Prompt version (tag, commit): 9c80396021a8106bfaeade9a1ea51b85152e951d

```
master+ REBASING ± git rebase --abort
It looks like git-am is in progress. Cannot rebase.
```

**Comment by laurentbardin** [comment]
For what it's worth, I tried an empty `git am` on my machine followed by `^C` and had the `REBASING` tag added, but `git rebase --abort` didn't complain and removed it correctly.

macOS 10.12.5, git 2.13.2, liquid prompt 9c80396021a8106bfaeade9a1ea51b85152e951d, GNU bash, version 4.4.12(1)-release (x86_64-apple-darwin16.4.0) (git and bash installed using macport).

**Comment by cburschka** [comment]
Just encountered the same issue. It looks like `git-am` tracks its state via an identical folder (`.git/rebase-apply`) to `git-rebase`.

git-rebase itself uses another simple check to distinguish its own state from git-am's: [The presence of a file named `.git/rebase-apply/applying`](https://github.com/git/git/blob/ed843436dd4924c10669820cc73daf50f0b4dabd/git-rebase.sh#L240).

Going to make a PR to add this case.

**Comment by Rycieos** [comment]
This is fixed in [v2.0.0-beta.1](https://github.com/nojhan/liquidprompt/releases/tag/v2.0.0-beta.1): [`_lp_git_head_status()`](https://liquidprompt.readthedocs.io/en/v2.0.0-beta.1/functions/data/vcs.html#lp_git_head_status). See the link for documentation.

The implementation details:
https://github.com/nojhan/liquidprompt/blob/6e7f4c77044bea7ab92970e820997601926bcbea/liquidprompt#L1392-L1417

Thanks for the bug report, and thanks to @cburschka for the partial implementation. I based my implementation on the official Git prompt: https://github.com/git/git/blob/3cf59784d42c4152a0b3de7bb7a75d0071e5f878/contrib/completion/git-prompt.sh#L446-L469
